LDAR_Sim/src/file_processing/output_processing/summary_outputs.py

```diff
@@ -115,7 +115,8 @@ def generate_emissions_summary(directory: str, outputs_mapper: SummaryOutputMapp
         how="outer",
     )
     # Fill NaN values with zeros
-    merged.fillna(0, inplace=True)
+    with pd.option_context("future.no_silent_downcasting", True):
```

Comment:
I suggest doing something like:
df.astype(float).fillna(value)
df.infer_objects(copy=False).fillna(value)
instead of muting the warning. In theory this only mutes the warning during the time the context manager is active so it might be fine, but actually solving the error is always better.
Can you attach the actual error you got here if my suggest doesn't work so we can dive deeper.

Reply:
Error:
![image](https://github.com/LDAR-Sim/LDAR_Sim/assets/43421303/cb2beb64-4c66-4047-b292-f240e5f5bc73)